polar_route/exceptions.py

```diff
@@ -2,35 +2,35 @@
     This module contains the custom exceptions for the polar_route package.
 """
 
-class WayPointOutOfBounds(Exception):
+class WayPointOutOfBoundsError(Exception):
     "Raised when the waypoint is out of bounds of the mesh"
 
     def __init__(self, waypoint, message="Waypoint is out of bounds of the mesh"):
         self.message = "Waypoint at " + str(waypoint) + " is out of bounds of the mesh"
         super().__init__(self.message)
 
-class NoRouteFound(Exception):
+class NoRouteFoundError(Exception):
     "Raised when a route cannot be found between two waypoints"
 
     def __init__(self, message="No route found"):
         self.message = message
         super().__init__(self.message)
 
-class InaccessibleWaypoint(Exception):
+class InaccessibleWaypointError(Exception):
     "Raised when a waypoint is inaccessible"
 
     def __init__(self, waypoint, message="Waypoint is inaccessible"):
         self.message = "Waypoint at " + str(waypoint) + " is inaccessible"
         super().__init__(self.message)
 
-class RouteCouldNotSmooth(Exception):
+class RouteCouldNotSmoothError(Exception):
```

Comment:
This one might be more succinct as `RouteSmoothingError` which I think conveys the same information?